warehouse/query-core/src/main/java/datawave/query/jexl/visitors/ValidPatternVisitor.java

```diff
@@ -126,4 +128,24 @@ public void parseAndPutPattern(JexlNode node) {
             }
         }
     }
+    
+    // Descend through these nodes
+    @Override
+    public Object visit(ASTJexlScript node, Object data) {
```

Comment:
I don't think you need to override these methods as the short circuit base visitor does not block descending through these.

Reply:
For some reason I was thinking BaseVisitor blocked but must have ben an oversight on my part.